sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -0,0 +1,185 @@
+-- Basic tests for intervals
+--SET spark.sql.parser.ansi.enabled=true
+
+select
+  '1' second,
+  2  seconds,
```

Comment:
`seconds` needs to be a reserved word in the ANSI mode; otherwise, this could be either alias or interval unit? Is my understanding right?

Reply:
Regardless of whether seconds reserved or not, this pr currently regards the statement as interval. Anyway, IMO seconds and the other units should be reserved in the ANSI mode.